schema.sql

```diff
@@ -79,3 +79,12 @@ CREATE TABLE visits (
   FOREIGN KEY (animal_id) REFERENCES animals(id),
   FOREIGN KEY (vet_id) REFERENCES vets(id)
 );
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX animal_id_index ON visits (animal_id);
+
+CREATE INDEX vet_id_index ON visits (vet_id);
+
+CREATE INDEX email_index ON owners (email);
```

Comment:
[OPTIONAL] I suggest making the index on the case-insensitive version of the email. this won't improve the query that was reported as problematic in the requirements for this project. but we know that emails shouldn't be case sensitive and so this will make it easier to look for emails😉. you can reflect that in the index using the query below😉.

```suggestion
CREATE INDEX email_index ON owners ((lower(email)));
```